### PR TITLE
Added code for divergent summation series

### LINKED ISSUE #14579: Can't solve limit in sqrt
```
>>> from sympy import *
>>> x=symbols('x')
>>> Sum(sqrt(x), (x,1,oo)).doit()
Sum(sqrt(x), (x, 1, oo))
```
where the answer should have been ```oo```(related to https://github.com/sympy/sympy/issues/14393)

### PR DESCRIPTION
**Pull request by dnabanita7**
Sum(sqrt(n), (n, 1, oo)) didn't calculate or simplified previously.It gave only its corresponding symbolic representation.As discussed in the issue thread,positive divergent series should return oo. Fixed in line 203-204.
Fixes #14579

* series 
     * added check for positive divergent series in doit() of summation.py

### CODE REVIEW

**Comment by oscargus** [comment]
Please correct the code and add some tests.

**Comment by oscargus** [comment]
Btw, what will `Sum(-sqrt(n), (n, 1, oo))` give?

**Comment by Naba7** [comment]
@oscargus can you please give a look

**Comment by asmeurer** [comment]
Please run the code locally before committing it. This isn't even valid Python. It's best to avoid using GitHub's web editor. 

Aside from that, there are some other issues with what your logic here:

- You are assuming the summation is infinite. 
- The is_positive check will fail unless the user sets the summation variable as positive. It would probably be best to substitute it with a positive Dummy before performing the check. 
- This logic should go in `eval_sum`.

**Comment by ShubhamKJha** [comment]
@Naba7 the Python equivalent of `!` is `not`. Besides, you should check your code before even committing.
The best way to have a development workflow is:
1. Clone the repo in a specific directory.
2. Use python from that directory. You shouldn't install `SymPy`, you can `import sympy` from there.
3. Test if your solution fixes the given issue with correct outputs. Most of your errors gets fixed this way.
4. Run tests for that specific feature. You can find help [here](https://github.com/sympy/sympy/wiki/Running-tests).
5. Only then commit and push.

**Comment by Naba7** [comment]
Ok,thanks @ShubhamKJha . I am new to git and its workflow.But I will learn this and implement it from next time onwards

File: sympy/concrete/summations.py [hunk 1]
```diff
@@ -946,12 +945,19 @@ def eval_sum(f, limits):
         return eval_sum_direct(f, (i, a, b))
     if isinstance(f, Piecewise):
         return None
+    #If the number of terms is infinite and series is convergent
+    if !(f.is_convergent) and expr.subs(1,a+1)is_positive:
```
Comment (supreet11agrawal) [inline sympy/concrete/summations.py]: It doesn't work this way. You should write `if not`

Comment (Naba7) [reply sympy/concrete/summations.py]: yes.I am changing it

File: sympy/concrete/summations.py [hunk 2]
```diff
@@ -200,7 +199,7 @@ def doit(self, **hints):
                 else:
                     return self.func(f, *self.limits[n:])
             f = newf
-
+
```
Comment (oscargus) [inline sympy/concrete/summations.py]: Spaces that will make the code quality test fail.

File: sympy/concrete/summations.py [hunk 3]
```diff
@@ -946,12 +945,19 @@ def eval_sum(f, limits):
...
+    elif not(f.is_convergent) and expr.subs(i,a+1).is_negative:
+        return -oo
+
     # Try to do it symbolically. Even when the number of terms is known,
     # this can save time when b-a is big.
     # We should try to transform to partial fractions
-    value = eval_sum_symbolic(f.expand(), (i, a, b))
-    if value is not None:
-        return value
+    if b != oo:
```
Comment (oscargus) [inline sympy/concrete/summations.py]: There are series that goes to oo that will converge. (And from -oo).

File: sympy/concrete/summations.py [hunk 4]
```diff
@@ -946,12 +945,19 @@ def eval_sum(f, limits):
         return eval_sum_direct(f, (i, a, b))
     if isinstance(f, Piecewise):
         return None
+    #If the number of terms is infinite and series is convergent
+    if not(f.is_convergent) and expr.subs(1,a+1)is_positive:
```
Comment (oscargus) [inline sympy/concrete/summations.py]: should this be i? What about a series starting from -2?

File: sympy/concrete/summations.py (hunk 2 above)
Comment (Naba7) [reply sympy/concrete/summations.py]: done

File: sympy/concrete/summations.py (hunk 4 above)
Comment (Naba7) [reply sympy/concrete/summations.py]: Yes,I have changed it now.Thank you

File: sympy/concrete/summations.py (hunk 3 above)
Comment (Naba7) [reply sympy/concrete/summations.py]: thank you.I have made necessary changes

**Pull request closed without merging**